roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -179,6 +179,9 @@ masterPublicURL: {{ openshift.master.public_api_url }}
 networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
+  clusterNetworks:
```

Comment:
I don't think it makes sense to set both the old and the new values; we should set the old values for old installs and the new values for new installs. I don't know how to do that but there's clearly some ability to conditionalize things based on the lines below, and there must be some example in some file somewhere of conditionalizing on version...

Reply:
@danwinship There was a bug with the multiple CIDR addition that clayton fixed by adding the old values. I tried to avoid it but now you need both the new and old values for the system to work and the first entry in clusterNetworks needs to match clusterNetworkCIDR/hostsubnetlength

Reply:
```
networkConfig:
{% if openshift.common.version_gte_3_7 | bool %}
  - cidr: {{ openshift.master.sdn_cluster_network_cidr }}
    hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
{% else %}
  clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
  hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
{% endif %}
...
```

Reply:
nm, didn't see that you had to have both.

Reply:
If it's only valid in 3.7 and newer we should conditionalize the generation.